Gopkg.toml

```diff
@@ -57,10 +65,15 @@ required = [
   name = "k8s.io/kube-openapi"
   revision = "0cf8f7e6ed1d2e3d47d02e3b6e559369af24d803"
 
-[prune]
-  go-tests = true
-  non-go = true
+[[constraint]]
+  name = "github.com/onsi/gomega"
+  version = "1.4.3"
 
-  [[prune.project]]
-    name = "k8s.io/code-generator"
-    non-go = false
+[[constraint]]
+  name = "github.com/onsi/ginkgo"
+  version = "1.7.0"
+
+# Apply workaround from https://github.com/golang/dep/issues/1799
```

Comment:
not part of PR ?

Reply:
Maybe comment is misleading. This issue is still open and without this constraint, it fails fetching ginkgo/gomega. I will reword the comment to avoid confusion.

Reply:
Addressed in b83b9b3.